.gitignore

```diff
@@ -8,5 +8,5 @@ tmp
 test/logs
 coverage
 .sonar
-lib
+!lib
```

Comment:
do not add `lib/` folder!!

Reply:
sorry. was just a fix for our broken circleci script :)